src/ghost/ghost-post.service.ts

```diff
@@ -113,4 +129,57 @@ export class GhostPostService {
 
         return await this.postService.deleteByApId(apId, account);
     }
+
+    async createGhostPost(
```

Comment:
It looks like there is going to be multiple inserts happening (1 for the post, 1 for the mapping), we should probably use a transaction for these

Reply:
Added error handling for ghost post mapping insert failures. We'll clean up the post if mapping insertion fails. Changes here: https://github.com/TryGhost/ActivityPub/pull/1054/commits/bcb40cdf704bd51c14b2c7d1e9693d67e1e9ef82